registry_quickstart/administrators/system_configuration.adoc

```diff
@@ -128,6 +131,16 @@ Restart the origin service to update the running configuration.
 $ sudo docker restart origin
 ----
 
+== Using Service Account Tokens for Authentication
+
+Typically long-lived, token-based authentication is desired. As an alternative
+to using user session tokens that expire, users may use
+link:../../admin_guide/service_accounts.html[service account tokens] to
+authenticate with docker. This is particularly useful when integrating automation.
+See
+link:../developers.html#using-service-account-tokens-for-docker-login[quickstart developer guide]
```

Comment:
same here:
xref:../developers.adoc#using-service-account-tokens-for-docker-login[quickstart developer guide]

s/see/see the